azure-pipelines.yml

```diff
@@ -8,7 +8,10 @@
 
 name: $(date:yyyyMMdd)$(rev:.r)
 
-pool: MsGraphBuildAgentsWindows
+pool:
+  vmImage: 'windows-latest'
+  steps:
+  - script: echo hello from Windows
```

Comment:
we don't have access to this [agent pool](https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/pools-queues?view=azure-devops&tabs=yaml%2Cbrowser) that is used for the original repo, so i used the default Azure Pipelines pool here with a Windows vmImage.